apps/meteor/server/models/raw/PersistentQueue.ts

```diff
@@ -0,0 +1,20 @@
+import type { Db, IndexDescription } from 'mongodb';
+import type { IPersistentQueueModel } from '@rocket.chat/model-typings';
+import type { IPersistentQueue } from '@rocket.chat/core-typings';
+
+import { BaseRaw } from './BaseRaw';
+
+export class PersistentQueueModel extends BaseRaw<IPersistentQueue> implements IPersistentQueueModel {
```

Comment:
If this is to add indexes to the collection mongomessagequeue uses, i vaguely remember it was `_queue`

Additional, if it's the same collection and you're adding indexes here, you can remove the index creation step on the queue-worker microservice

Reply:
you're right this is to setup the indexes! 

But we're not using the same collection since in one moment both this new package and the service used by Omnichannel will live at the same time. 

An additional PR could be made to update Omnichannel to this package and remove the other service.